README.markdown

```diff
@@ -144,6 +144,7 @@ Supported options:
 
 * If you pass `--ts_proto_opt=context=true`, the Twirp services will have a Go-style `ctx` parameter, which is useful for tracing/logging/etc. if you're not using node's `async_hooks` api due to performance reasons.
 * If you pass `--ts_proto_opt=forceLong=long`, all 64 bit numbers will be parsed as instances of `Long` (using the [long](https://www.npmjs.com/package/long) library). Alternatively, if you pass `--ts_proto_opt=forceLong=string`, all 64 bit numbers will be outputted as strings.
+* If you pass `--ts_proto_opt=forceJsonBase64Bytes=true`, the `Message.fromJSON` and `Message.toJSON` methods will treat bytes as base64 encoded strings. This currently relies on `window.atob` and `window.btoa` being available.
```

Comment:
Hm, kind of thinking, does this need to be an option? I.e. could we just always output this code? I don't think that would break anyone's code/behavior today, because this is just not working at all AFAIU.

Granted, it's not going to work on node today, but at some point someone who is running this on node will file an issue, and we can fix the code then to support both node+browser...without any flags, I think?

Reply:
I just assumed that multiple encodings for `bytes` could be chosen, but checking the [docs](https://developers.google.com/protocol-buffers/docs/proto3#json) base64 is the one and only encoding for the JSON mapping. I'll remove the option.

Reply:
Great!